components/EditCarForm/index.js

```diff
@@ -0,0 +1,166 @@
+            id="widthInput"
+            defaultValue={activeCar["Width (mm)"]}
+          />
+          (mm)
+        </label>
+        <label htmlFor="widthWithMirrorsInput">
+          Breite inkl. Spiegel:
+          <input
+            type="number"
+            name="Width including mirrors (mm)"
+            id="widthWithMirrorsInput"
+            defaultValue={activeCar["Width including mirrors (mm)"]}
+          />
+          (mm)
+        </label>
+        <label htmlFor="heightInput">
+          Höhe:
+          <input
+            type="number"
+            name="Height (mm)"
+            id="heightInput"
+            defaultValue={activeCar["Height (mm)"]}
+          />
+          (mm)
+        </label>
+        <label htmlFor="emptyWeightInput">
+          Leergewicht:
+          <input
+            type="number"
+            name="Weight Empty (kg)"
+            id="emptyWeightInput"
+            defaultValue={activeCar["Weight Empty (kg)"]}
+          />
+          (kg)
+        </label>
+        <label htmlFor="maxWeightInput">
+          Max. Gewicht:
+          <input
+            type="number"
+            name="Max Weight (kg)"
+            id="maxWeightInput"
+            defaultValue={activeCar["Max Weight (kg)"]}
+          />
+          (kg)
+        </label>
+      </fieldset>
+      <fieldset>
+        <p>Weitere Daten:</p>
+        <label htmlFor="drive">
+          Antrieb:
+          <input
+            type="text"
+            name="Drive"
+            id="drive"
+            defaultValue={activeCar.Drive}
+          />
+        </label>
+        <label htmlFor="model-year">
+          Modelljahr:
+          <input
+            type="number"
+            name="Model Year"
+            id="model-year"
+            defaultValue={activeCar["Model Year"]}
+          />
+        </label>
+        <label htmlFor="engineDisplacementInput">
+          Hubraum:
+          <input
+            type="text"
+            name="Engine Displacement (ccm)"
+            id="engineDisplacementInput"
+            defaultValue={activeCar["Engine Displacement (ccm)"]}
+          />
+          (ccm)
+        </label>
+        <label htmlFor="transmission">
+          Getriebe:
+          <input
+            type="text"
+            defaultValue={activeCar.Transmission}
+            name="Transmission"
+            id="transmission"
+          />
+        </label>
+        <label htmlFor="numberOfGearsInput">
+          Max Speed (km/h):
+          <input
+            type="number"
+            name="Number Of Gears"
+            id="numberOfGearsInput"
+            defaultValue={activeCar["Number of Gears"]}
+          />
+        </label>
+        <label htmlFor="maxSpeedInput">
+          Max Speed (km/h):
+          <input
+            type="number"
+            name="Max Speed (km/h)"
+            id="maxSpeedInput"
+            defaultValue={activeCar["Max Speed (km/h)"]}
+          />
+        </label>
+      </fieldset>
+
```

Comment:
Id ont think that is possible, because i have over 50 keys in may activeCar prop. If i map over all I have props that are not needed and they aren't together in groups. If I try to group them in arrays like const dimensions= [Height, Width, Weight...] i cant get the default values. And if i gruop them like that [{group: "dimensions", Width: activeCar.width, Height: activeCar.height}] I have to write each label/input field one by one which doesn't have any benefit. 
If there is another way I would be glad to know it.

Reply:
Here is an isolated, not so easy to understand, prototype that does what you need :) 

https://codesandbox.io/s/mapping-inputs-3utq1t?file=/src/App.js

Reply:
Thank you, needed some little changes but now it works fine!